src/site/content/en/blog/compress-images-avif/index.md

```diff
@@ -0,0 +1,348 @@
+---
+layout: post
+title: Using AVIF to compress images on your site
+authors:
+  - jaikk
+  - wtc
+hero: image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/0jtMLyN4T7C7UazJCYEH.jpg
+alt: The AVIF logo
+description: |
+  Serving desktop-sized images to mobile devices can use 2–4x more data than
+  needed. Instead of a "one-size-fits-all" approach to images, serve different
+  image sizes to different devices.
+date: 2021-06-04
+tags:
+  - performance
+  - blog
+---
+
+We
+[frequently write](/fast/#optimize-your-images)
+about the bloat on websites from images,
+and tools like
+[Lighthouse](/optimize-vitals-lighthouse/)
+highlight when image loading is having a negative impact on user experience,
+such as increasing load time,
+or taking bandwidth away from more important resources.
+One way to fix this is to use modern compression to reduce the file size of images,
+and a new option for web developers is the
+[AVIF image format](https://aomediacodec.github.io/av1-avif/).
+This blog post talks about recent updates to open source tooling for AVIF,
+introduces the libaom and libavif encoding libraries,
+and includes a tutorial for using these libraries to encode AVIF images efficiently.
+
+AVIF is an image format based on the AV1 video codec,
+and standardized by the
+[Alliance for Open Media](http://aomedia.org).
+AVIF offers significant compression gains over other image formats like JPEG and WebP.
+While the exact savings will depend on the content, encoding settings, and quality target,
+[we](https://jakearchibald.com/2020/avif-has-landed/) and
+[others](https://netflixtechblog.com/avif-for-next-generation-image-coding-b1d75675fe4)
+have seen >50% savings vs. JPEG.
+
+<div class="w-columns">
+<figure class="w-figure">
+  {% Img
+  src="image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/RRE6Gv9EQ986PLHTKI4g.png",
+  alt="The image using AVIF",
+  width="800",
+  height="1067" %}
+  <figcaption class="w-figcaption">
+  1120 x 840 AVIF at 20,984 bytes
+</figure>
+<figure class="w-figure">
+  {% Img
+  src="image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/WBvRmc1oH6tWR3WfqZ5y.jpg",
+   alt="The image using JPEG",
+   width="800",
+   height="1067" %}
+  <figcaption class="w-figcaption">
+  1120 x 840 JPEG at 29,775 bytes
```

Comment:
This image is unnecessarily bad and misrepresents JPEG. Please hold off on publishing this until I have reached out to the authors (but don’t let me block you on any other worker).